script/Ciel/Day 3/03_01_CIEL03_3_2B.txt

```diff
@@ -350,7 +351,8 @@ I was lying to myself when I said I couldn't think straight.
 [sha:0d72519dbf38b12672c3c6cdd9ab88ecda651630]{
 -- Page 149, Offset 5230.
 -- だってさっきからそうしたくて、俺は何もしなかった。
-I've wanted to this whole time, I just haven't.
+// Is this closer to the intended meaning that the comment in the previous line mentioned? Need a JP check.
+I could think this whole time, I just haven't.
```

Comment:
Original is correct, sentence conveys intent not ability. (though this change makes more sense from an english perspective, agreed).